File: lv1/숫자 문자열과 영단어.py
obj = (
  ["zero", 0], 
  ["one", 1], 
  ["two", 2], 
  ["three", 3], 
  ["four", 4], 
  ["five", 5], 
  ["six", 6], 
  ["seven", 7], 
  ["eight", 8], 
  ["nine", 9]
)

def solution(s):
  for word, num in obj:
    s = s.replace(word, str(num))
  return int(s)

File: 카카오인턴십/lv1/숫자 문자열과 영단어.js
const obj = {
  0: "zero",
  1: "one",
  2: "two",
  3: "three",
  4: "four",
  5: "five",
  6: "six",
  7: "seven",
  8: "eight",
  9: "nine"
}
function solution(s) {
  let str = s;
  const values = Object.values(obj);
  const keys = Object.keys(obj);
  values.forEach((e, i) => {
    str = str.replaceAll(e, keys[i])
  })
  var answer = str;
  return parseInt(answer);
}